dist/lob-api-bundled.yml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   title: Lob
-  version: 1.19.23
```

Comment:
looks like this was edited manually for some reason without going through the steps in the README. there is no existing tag on 1.19.22 or .23 so we're good